plugins/cmp/default.nix

```diff
@@ -15,6 +15,10 @@ lib.nixvim.plugins.mkNeovimPlugin {
   maintainers = [ lib.maintainers.GaetanLepage ];
 
   description = ''
+    A completion engine for Neovim written in Lua, designed to be fast and extensible.
+
+    ---
```

Comment:
Is this following an established convention elsewhere in nixvim? I'm not convinced a `<hr>` is really necessary immediately before a `<h2>`